ChangeLog.md

```diff
@@ -9,6 +9,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Fixed
 - Deprecation warning for `datetime.utcnow()`
 
+### Added
+- Added pre-commit hooks and Github CI action for code formatting and linting.
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore. -->
**💣 chore:** I think this should be under `Unreleased` instead of under `v0.3.11`

Reply:
Thanks for resolving 👍